### PR TITLE
#163794642 Define followers table model

### PR DESCRIPTION
**Pull request by Ndifreke**
**What does this PR do ?**
* Creates Followers table model

**Description of Task to be completed ?**
* Create an association between Authors and Followers

**What are the relevant pivotal tracker story ?**
[#163794642](https://www.pivotaltracker.com/n/projects/2240406/stories/164215839)

### CODE REVIEW

**Review by obasajujoshua31 — APPROVED** [review]

**Review by eokwukwe — APPROVED** [review]

**Review by mrkingz — APPROVED** [review]

**Review by benfluleck — CHANGES_REQUESTED** [review]
Please resolve the vulnerability fix on this branch. Thanks @Ndifreke

File: server/tests/models/followers.spec.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import chai from "chai";
+import chaiAsPromise from "chai-as-promised";
+import database from "../../models";
```
Comment (benfluleck) [inline server/tests/models/followers.spec.js]: Please resolve these paths with the new syntax

**Review by benfluleck — CHANGES_REQUESTED** [review]

File: server/tests/models/followers.spec.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    authorId,
+    followerId
+  });
+};
+
+beforeEach(async () => {
+  await sequelize.sync({ force: true });
+});
+
+describe("Fellowers", () => {
```
Comment (benfluleck) [inline server/tests/models/followers.spec.js]: What is Fellowers ? huh 🤔

File: server/tests/models/followers.spec.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  it("should create a fellowers table", async () => {
+    const { followerId, authorId } = await fellowersDependencies();
+    const fellowers = await Followers.create({ followerId, authorId });
```
Comment (benfluleck) [inline server/tests/models/followers.spec.js]: fellowers huh?

File: server/tests/models/followers.spec.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    expect(fellowers).to.not.be.null;
+  });
+
+  it("should remove a follower when he is deleted in Users table", async () => {
+    const { followerId, authorId, fellow } = await fellowersDependencies();
+    await Followers.create({ followerId, authorId });
+    await fellow.destroy();
```
Comment (benfluleck) [inline server/tests/models/followers.spec.js]: Continue to fellow

File: server/models/followers.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+      type: DataTypes.UUID,
+      allowNull: false
+    },
+    followerId: {
+      type: DataTypes.UUID,
+      allowNull: false
+    }
+  };
+
+  const Followers = sequelize.define("Followers", FellowersSchema, flags);
```
Comment (benfluleck) [inline server/models/followers.js]: FellowersSchema?

File: server/tests/models/followers.spec.js [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+chai.use(chaiAsPromise);
+const { Users, Followers, sequelize } = database;
+const { expect } = chai;
+const user1 = userData[0];
+
+const followersDependencies = async () => {
+  const authorsData = Object.assign({}, user1);
+  authorsData.email = "obj@email.com";
+  const author = await Users.create(authorsData);
+  const fellow = await Users.create(user1);
```
Comment (Ndifreke) [inline server/tests/models/followers.spec.js]: This is bad for me, I need to pay more attention

File: server/tests/models/followers.spec.js [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+    authorId,
+    followerId
+  });
+};
+
+beforeEach(async () => {
+  await sequelize.sync({ force: true });
+});
+
+describe("followers", () => {
```
Comment (benfluleck) [inline server/tests/models/followers.spec.js:30]: `Followers Model `

**Review by benfluleck — APPROVED** [review]

**Pull request merged**